rules/zend2-rocket/tests/Rector/ClassMethod/RenderZendViewRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
```

Comment:
Always use namespace to keep tests local-scoped

Reply:
see https://github.com/rectorphp/rector/pull/2882/commits/d752cbc576d6c2f3daf31184ebd9fd9f39d86f04

Reply:
The codebase which should be refactored is also global-scoped. Do you recommend to keep the test local-scoped anyways?

Reply:
If that's not crutial for the rule (e.g. in case of Under_Score_Name to namespaced one), then keep tests always namespaced.